assets/javascripts/modules/autoComplete.js

```diff
@@ -18,7 +18,7 @@ Auto complete html markup:
            autocomplete="off"
            spellcheck="false"
            required
-           data-suggestions="countries"
+           data-suggestions="{{OPTIONAL: id of select input to use as suggestions source}}"
```

Comment:
this `data-suggestions` attribute is used for validation purposes [here](https://github.com/hmrc/assets-frontend/blob/master/assets/javascripts/validation/customValidations.js#L12) (needs documenting)